src/app/components/store-products/store-products.component.ts

```diff
@@ -8,4 +16,35 @@ import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class StoreProductsComponent {
-}
+  readonly search: FormGroup = new FormGroup({ searchMarket: new FormControl() });
+
+  
+    readonly store$: Observable<StoreModel[]|any> = this._activatedRoute.params.pipe(
+      switchMap((data) => {
+       return this._storesService.getOneStore(data['storeId'])}
+     ),tap(()=>{
+      console.log(this.store$)
```

Comment:
It's look strange, variable call itself, in tap you have current data co: tap((data) => console.log(data)) or tap(console.log)

And it's best not to keep console.log